### PR TITLE
Remove FileImport wrapper class

### PR DESCRIPTION
**Pull request by alexsoble**
# Notes 

+ Taking a crack at #1146 to improve the legibility and error-resiliency of the data import process by removing the `FileImport` class.  `FileImport` "wraps" a bunch of other import classes and obscures the flow of control those classes use to import data.

### CODE REVIEW

File: app/importers/file_importers/downloader.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+class Downloader
```
Comment (kevinrobinson) [inline app/importers/file_importers/downloader.rb]: Is this a `CSVDownloader`?  It looks like it expects there be a `transformer` and that has a method `pre_cleanup_csv_size` etc.  Adding a comment about what the class is for could help.

File: app/importers/file_importers/downloader.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    data = @transformer.transform(file)
+
+    CleanupReport.new(
+      @log, @remote_file_name, @transformer.pre_cleanup_csv_size, data.size
+    ).print
+
+    return data
+  end
+
+  def download_file
```
Comment (kevinrobinson) [inline app/importers/file_importers/downloader.rb]: Is this intended to be public or private?

**Comment by kevinrobinson** [comment]
@alexsoble This looks good to me!

File: app/importers/file_importers/downloader.rb (hunk 1 above)
Comment (alexsoble) [reply app/importers/file_importers/downloader.rb]: @kevinrobinson Good points, will implement.

File: app/importers/file_importers/downloader.rb (hunk 2 above)
Comment (alexsoble) [reply app/importers/file_importers/downloader.rb]: @kevinrobinson — Private, I'm thinking `get_data` should be the only public method. Will add that.

**Pull request merged**